.gitlab-ci.yml

```diff
@@ -81,6 +118,27 @@ build:ts:
   needs: []
   stage: build
 
+build:sls:package:
+  <<: *tmpl_branch
+  image: $INTEGRATION_DOCKER_IMAGE
+  stage: build
+  variables:
+    <<: *tmpl_branch_variables
+    <<: *tmpl_deploy_variables
+  artifacts:
+    expire_in: 2 weeks
+    paths:
+      - '.serverless'
+  script:
+    - export SQUIDEX_APP_NAME=${CI_EXTERNAL_PULL_REQUEST_IID}
+    - export SQUIDEX_CLIENT_ID=$SQUIDEX_TEST_CLIENT_ID
+    - export SQUIDEX_CLIENT_SECRET=$SQUIDEX_TEST_CLIENT_SECRET
+    - source ci/env-setup.sh
+    - yarn rebuild # Duplicated for performance
```

Comment:
this is to rebuild the FE app etc? maybe we should make this dependant on the other build job and have it use its artifact?

Reply:
No, this is some PNP stuff. We wouldn't duplicate these commands if the machine allocation took less time. In our tests it was upwards of 30 seconds for 10 yarn rebuild